From f1b3b2296d4a399688b216172d5a999080d5e001 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Wed, 04 Jul 2018 15:45:51 +0300
Subject: [PATCH] board: fix board name for compliance with tbot

The actual board name as provided by tbot is dali-asic-tbot.
Accept more *asic* board names.

Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/board/habanalabs/common/ip_pdu b/board/habanalabs/common/ip_pdu
index f031064..8169e3f 100755
--- a/board/habanalabs/common/ip_pdu
+++ b/board/habanalabs/common/ip_pdu
@@ -10,7 +10,7 @@
 
 pdu_cred="--user=snmp --password=1234"
 case "$board" in
-*asic)
+*asic*)
 	pdu_uri=http://192.168.10.120
 	pwr_id=1
 	;;
diff --git a/board/habanalabs/common/remote_power b/board/habanalabs/common/remote_power
index b909d95..ffddbd9 100755
--- a/board/habanalabs/common/remote_power
+++ b/board/habanalabs/common/remote_power
@@ -11,7 +11,7 @@
 fi
 
 case "$board" in
-*asic)
+*asic*)
 	;;
 *onix)
 	;;
